metricbeat/module/docker/memory/memory.go

```diff
@@ -72,7 +72,9 @@ func (m *MetricSet) Fetch(r mb.ReporterV2) error {
 
 	memoryStats := m.memoryService.getMemoryStatsList(stats, m.dedot)
 	if len(memoryStats) == 0 {
-		return fmt.Errorf("No memory stats data available")
```

Comment:
Can we change it to debug? I am not quite sure if was put for a reason initially

Reply:
I'm adding a logger to the metricset (currently, it does not log at all).